diff --git a/node_modules/@capacitor/android/capacitor/build.gradle b/node_modules/@capacitor/android/capacitor/build.gradle
index 7dba8ec..a1b52e9 100644
--- a/node_modules/@capacitor/android/capacitor/build.gradle
+++ b/node_modules/@capacitor/android/capacitor/build.gradle
@@ -68,8 +68,8 @@ android {
         lintConfig file('lint.xml')
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_21
-        targetCompatibility JavaVersion.VERSION_21
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
     publishing {
         singleVariant("release")